### PR TITLE
mogrify: add page

### PR DESCRIPTION
**Pull request by wefhy**

### CODE REVIEW

File: pages/common/mogrify.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Mogrify
+
+> Tool that allows making operations on multiple image files ie. resizing, cropping, flipping, adding effects.
```
Comment (sbrl) [inline pages/common/mogrify.md]: Perhaps mention that it alters the original image (i.e. as compared with `convert`, which saves a new file)?

File: pages/common/mogrify.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Resize all jpg images in the folder to 50% of initial size:
```
Comment (sbrl) [inline pages/common/mogrify.md]: "of initial size" -> "of their initial size"

File: pages/common/mogrify.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert all png images in the folder to jpg:
+
+`mogrify -format jpg *.png`
+
+- Halve the saturation for all files:
+
+`mogrify -modulate 100,50 *`
+
+- Double the brightness:
```
Comment (sbrl) [inline pages/common/mogrify.md]: ....of all files?

File: pages/common/mogrify.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Mogrify
```
Comment (agnivade) [inline pages/common/mogrify.md]: s/Mogrify/mogrify.

We always start a command with lower case.

File: pages/common/mogrify.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+> Tool that allows making operations on multiple images ie. resizing, cropping, flipping, adding effects. Replaces the original file.
```
Comment (agnivade) [inline pages/common/mogrify.md]: Would `viz.` be a better replacement for `ie.` ?

File: pages/common/mogrify.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- Resize all images starting with DSC to 800x600:
+
+`mogrify -resize 800x600 DSC*`
+
+- Convert all png images in the folder to jpg:
+
+`mogrify -format jpg *.png`
+
+- Halve the saturation for all files in the folder:
```
Comment (agnivade) [inline pages/common/mogrify.md]: When you say all files, it naturally raises the question - what files are supported ? 

I think you can add a second line in the heading section mentioning - "Supports image formats - jpg, png <fill others> ".

And in this description, instead of "all files", change it to "all supported files".

File: pages/common/mogrify.md (hunk 5 above)
Comment (sbrl) [inline pages/common/mogrify.md]: What does `viz.`

File: pages/common/mogrify.md (hunk 6 above)
Comment (sbrl) [inline pages/common/mogrify.md]: [imagemagick supports practically any image file you can think of](https://hastebin.com/irawibipit).

Comment (agnivade) [inline pages/common/mogrify.md]: Cool, then I think "for all image files" is appropriate.

File: pages/common/mogrify.md (hunk 5 above)
Comment (agnivade) [inline pages/common/mogrify.md]: "namely". https://en.wikipedia.org/wiki/Viz.

File: pages/common/mogrify.md (hunk 4 above)
Comment (wefhy) [inline pages/common/mogrify.md]: Hmm, I changed it to capital after bot complained, I must have misread the error message.
If we always start with lowercase, maybe add it also to bot's check?

File: pages/common/mogrify.md (hunk 5 above)
Comment (wefhy) [inline pages/common/mogrify.md]: I would stay with `ie.`. I also didn't know what `viz.` means, it may be unclear for some people. I think, `ie.` fits there well.

Comment (agnivade) [inline pages/common/mogrify.md]: I recommended `viz.` because it is often used to cite examples or variants of something. And `ie.` is generally used to explain something in simpler terms.

But it is a minor thing, and if you guys think that `ie.` is well suited here, then I'm okay with it.

Comment (wefhy) [inline pages/common/mogrify.md]: OK, I'm not english native, if you think it fits better, I believe you, I'll also learn something ;)

Comment (agnivade) [inline pages/common/mogrify.md]: > OK, I'm not english native,

me neither :P

**Comment by agnivade** [comment]
I'll wait for @sbrl to give a final opinion on the `viz` vs `ie` thing before merging it.

File: pages/common/mogrify.md (hunk 5 above)
Comment (sbrl) [inline pages/common/mogrify.md]: I'd recommend either `i.e.` or `namely`, as an english native speaker :P

Comment (agnivade) [inline pages/common/mogrify.md]: Let's go with `ie` then. `namely` just sounds weird.

File: pages/common/mogrify.md (hunk 4 above)
Comment (waldyrious) [reply pages/common/mogrify.md]: @agnivade, adding that check needs to be done in [tldr-lint](https://github.com/tldr-pages/tldr-lint), right?

File: pages/common/mogrify.md (hunk 5 above)
Comment (waldyrious) [reply pages/common/mogrify.md]: Coming late to the discussion, but IMO we should avoid abbreviations if possible (and in any case, the spelling is `i.e.`, not `ie.`. I'll make a PR suggesting a change to `such as`, which IMO is the most beginner-friendly option :)
